sensorium/training/trainers.py

```diff
@@ -36,7 +36,7 @@ def standard_trainer(
     maximize=True,
     tolerance=1e-6,
     restore_best=True,
-    lr_decay_steps=3,
+    lr_decay_steps=5,
```

Comment:
@Nisone2000 why?

Reply:
I did this when I had the issue with the peak for the cluster centers still with the orientation equivariant model to get better results. Now I don't really need it anymore.